lib/addons/keda/index.ts

```diff
@@ -0,0 +1,143 @@
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi";
+import { setPath, createNamespace } from "../../utils";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface KedaAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Version of the helm chart to deploy
+     */    
+    version?: string,
```

Comment:
We use google code convention for typescript, so based on [this](https://google.github.io/styleguide/tsguide.html#member-property-declarations) please use ";" to separate individual members.
Minor.

Reply:
fixed